HTML+CSS+JS/README.md

```diff
@@ -73,4 +73,5 @@
 
     -   [Vertical Timeline](Timelines/VerticalTimeline)
     -   [Reading Progress Bar](ReadingProgressBar)
+    -   [Timeline component](TimeLine component)
```

Comment:
Here, is the error I am talking about.
You can even use dashes '-' to separate the words.

Reply:
**Tip**

For example, you make the new name `TimeLineComponent`
Then in the  HTML+CSS+JS/README.md file, you just need to replace the above-selected line with:
`- [TimeLineComponent](Timelines/TimeLineComponent)`
Though take care of indentation and also test it before pushing.  😁🙂

Reply:
Sorry for that. Thanks for help...next time I will do that things